index.md

```diff
@@ -109,7 +109,7 @@ Web Content Accessibility Guidelines
 <!-- .slide: data-background="images/background/01.svg" -->
 
 <p class="centerp">
-    <a href="#">Demo: Focus and tab order</a>
+    <a href="demos/focus_and_tab-order/index.html">Demo: Focus and tab order</a>
```

Comment:
There is a typo here. The `href` should be: "demos/focus_and_tab_order/index.html".

Reply:
Thanks - fixed!